src/main/java/domain/PrizeEnum.java

```diff
@@ -25,4 +26,14 @@ public long getHit() {
     public BigDecimal getPrizeAmt() {
         return prizeAmt;
     }
+
+    public static PrizeEnum of(long hit, boolean matchBonus) {
+        for (PrizeEnum prizeEnum : values()) {
+            if (prizeEnum.getHit() == hit) {
```

Comment:
```suggestion
            if (prizeEnum.hit == hit) {
```
본인의 필드이니 getter를 따로 안쓰고 접근이 가능할 것 같네요 👀